kotest-framework/kotest-framework-api/src/commonMain/kotlin/io/kotest/core/datatest/datatestconcurrent.kt

```diff
@@ -0,0 +1,42 @@
+package io.kotest.core.datatest
```

Comment:
I would rather these live in the same file so avoid import clashes. That way when you import forAll from this package, you can use it in both root scope and container scope in the same spec.